src/extension.ts

```diff
@@ -37,6 +37,9 @@ export async function activate(context: vscode.ExtensionContext) {
     }
     await memento.saveSchemaVersion(storage.schemaVersion);
   }
+  const testIDs = memento.getTestIDs();
```

Comment:
Would be worthwhile to add a note here for our future selves, explaining why we're doing this ❔

Reply:
👍